packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts

```diff
@@ -49,6 +49,7 @@ export type AccordionProps<Value = AccordionItemValue> = ComponentProps<Accordio
   /**
    * Callback to be called when the opened items change.
    */
+  // eslint-disable-next-line @fluentui/consistent-callback-type
```

Comment:
it would be good idea to also provide justification why is this  being disabled (for all occurrences in this PR).